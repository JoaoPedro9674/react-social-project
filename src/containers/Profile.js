import React, { Component } from 'react';

export class Profile extends Component {
  render() {
    return (
      <div>
        profile will go here!
      </div>
    );
  }
}
