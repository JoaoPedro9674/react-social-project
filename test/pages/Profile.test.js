import React from 'react';
import renderer from 'react-test-renderer';

import Profile from '../../src/pages/profile';

test('Profile should render correctly', () => {
    const component = renderer.create(<Profile />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
