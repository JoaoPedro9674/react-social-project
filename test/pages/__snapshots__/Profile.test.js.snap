// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile should render correctly 1`] = `
<div
  className="profile"
>
  <div
    className="row"
  >
    <div
      className="col-xs-offset-2 col-xs-8"
    >
      <h2>
        Profile settings!
      </h2>
      <input
        onChange={[Function]}
        type="text"
      />
    </div>
  </div>
</div>
`;
